crates/polars-stream/Cargo.toml

```diff
@@ -26,10 +26,11 @@ tokio = { workspace = true }
 
 polars-core = { workspace = true }
 polars-error = { workspace = true }
-polars-expr = { workspace = true }
-polars-mem-engine = { workspace = true, features = ["parquet"] }
+polars-expr = { workspace = true, features = ["dtype-full"] }
+# TODO: feature gate
```

Comment:
@nameexhaustion I temporarily activated everything, but this should be feature gated properly with the dependencies and withing the `polars-stream` code.